src/mongocrypt.h

```diff
@@ -1082,6 +1082,8 @@ typedef struct _mongocrypt_kms_ctx_t mongocrypt_kms_ctx_t;
  * If KMS handles are being handled synchronously, the driver can reuse the same
  * TLS socket to send HTTP requests and receive responses.
  *
+ * The returned KMS handle must not outlive `ctx`.
```

Comment:
```suggestion
 * The returned KMS handle does not outlive `ctx`.
```

`must not` sounds like it's a requirement on the user/caller (i.e. they need to destroy the KMS handle before destroying the context), but it's not, it's just a description of the lifetime relationship between these objects that libmongocrypt already provides (right?)

Reply:
Agreed. Thank you for the suggestion.